client/my-sites/post-share/index.jsx

```diff
@@ -123,7 +140,26 @@ class PostShare extends Component {
 	};
 
 	sharePost = () => {
-		this.props.sharePost( this.props.siteId, this.props.postId, this.state.skipped, this.state.message );
+		const {
+			postId,
+			siteId,
+			connections,
+		} = this.props;
+		if ( this.state.scheduledDate ) {
+			const servicesToPublish = difference(
+				connections.map( connection => connection.keyring_connection_ID ),
+				this.state.skipped
+			);
+			this.props.schedulePostShareAction(
```

Comment:
By having code inside a block, I can clearly see with intendations where is the else block and what to expect.
By using returns instead of blocks I have to wonder where the previous block started and I have to notice return to grok the fact that someone decided not to use else.
It is not immidiately clear for me how the condition ends. I think pattern of using return instead of else does not improve readability when there are only 2 conditions.
I agree that this is a sensible pattern when we have multilple conditions, but I don't think `else` keyword is so inherently bad that we should get rid of it.